recipe.yaml

```diff
@@ -0,0 +1,74 @@
+# `recipe.yaml` is the main configuration file for an MLflow Recipe.
+# Required recipe parameters should be defined in this file with either concrete values or
+# variables such as {{ INGEST_DATA_LOCATION }}.
+#
+# Variables must be dereferenced in a profile YAML file, located under `profiles/`.
+# See `profiles/local.yaml` for example usage. One may switch among profiles quickly by
+# providing a profile name such as `local` in the Recipe object constructor:
+# `r = Recipe(profile="local")`
+#
+# NOTE: All "FIXME::REQUIRED" fields in recipe.yaml and profiles/*.yaml must be set correctly
+#       to adapt this template to a specific classification problem. To find all required fields,
+#       under the root directory of this recipe, type on a unix-like command line:
+#       $> grep "# FIXME::REQUIRED:" recipe.yaml profiles/*.yaml
+#
+# NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
+#       files are properly formatted.
+
+template: "classification/v1"
```

Comment:
Xiangrui Meng suggested to change this to `recipe: "classification/v1"`. I am going to make the corresponding change in mlflow, example repo and regression-template repo now.